test/jest/helpers/utils.js

```diff
@@ -13,3 +12,14 @@ export const componentPropsCheck = (Component, testId, expectedProps, partialCom
 
   expect(propertiesForCompare[0]).toStrictEqual(resultExpectedProps);
 };
+
+export const commonClassCheck = (Class, data) => {
+  const classInstance = new Class(data);
+  const passedDataKeys = Object.keys(data || {});
+
+  it(`should have correct values if ${passedDataKeys[0] ? `only "${passedDataKeys[0]}"` : 'nothing'} passed`, () => {
+    Object.keys(classInstance).forEach(key => {
+      expect(classInstance[key]).toBe(passedDataKeys.includes(key) ? data[key] : undefined);
+    });
+  });
```

Comment:
Maybe will be better to change it to the following view and as a result, we will have a possibility to test a set of properties:

```suggestion
    Object.keys(classInstance).forEach((key, index) => {
        it(`should have correct values if ${passedDataKeys[index] ? `only "${passedDataKeys[index]}"` : 'nothing'} passed`, () => {
            expect(classInstance[key]).toBe(passedDataKeys.includes(key) ? data[key] : undefined);
        });
    });
```